docs/design/autoware-concepts/difference-from-ai-and-auto.md

```diff
@@ -0,0 +1,37 @@
+# How is Autoware Core/Universe different from Autoware.AI and Autoware.Auto?
+
+Autoware is the world's first "all-in-one" open-source software for self-driving vehicles.
+Since it was first released in 2015, there have been multiple releases made with differing underlying concepts, each one aimed at improving the software.
+
+## Autoware.AI
+
+[Autoware.AI](https://github.com/Autoware-AI/autoware.ai) is the first distribution of Autoware that was released based on ROS 1. The repository contains different packages to cover different aspects of autonomous driving technologies, including sensing, actuation, localization, mapping, perception, planning, and so on.
```

Comment:
As a non-native speaker, I felt the words `different` are a bit unclear.
For me, this is easier to understand.

```suggestion
[Autoware.AI](https://github.com/Autoware-AI/autoware.ai) is the first distribution of Autoware that was released based on ROS 1. The repository contains a wide variety of packages to cover various aspects of autonomous driving technologies, including sensing, actuation, localization, mapping, perception, planning, and so on.
```

Reply:
I note that the list seems to be all of things required for autonomous driving, so we could perhaps shorten the list to only 3-4 items or rephrase to say:

`The repository contains a wide variety of packages to cover the various aspects of autonomous driving technology - sensing, actuation, localization, mapping, perception and planning

Reply:
My alternative suggestion:

```suggestion
[Autoware.AI](https://github.com/Autoware-AI/autoware.ai) is the first distribution of Autoware that was released based on ROS 1. The repository contains a variety of packages covering different aspects of autonomous driving technologies - sensing, actuation, localization, mapping, perception and planning.
```

Reply:
Cool! :+1: